bash_aliases.sh

```diff
@@ -1,26 +1,36 @@
-
+#!/bin/sh
 # Venv ALiases                                                                   
-alias venv-list='locate --regex "bin/activate"$'        # Locate /bin/activate paths for both conda and venv                         
-alias seder='sed  's/bin\/activate$//' '  # Cannot remember
+alias venv-list='locate --regex "bin/activate"$'    # Locate /bin/activate paths for both conda and venv                         
+alias seder='sed  's/bin\/activate$//' '            # Cannot remember
 
 #alias python=python2.7 
 
 # Conda Aliases                                                                  
-alias conda-remove-env='conda env remove --name $1'                             
-alias conda-create-env='conda create --name $1'                                 
-alias conda-lsp='conda list --name $1' # List conda packages                                          
-alias conda-ls='conda env list'        # List conda environments                                                  
-                                                                                 
-#Git Aliases                                                                     
-alias git-graph='git log --graph --oneline'                      # A simpler heuristic                                   
-alias git-show-lastn=' bash /home/meow/scripts/gitshowcom.sh $1' # Look behind at the nth commit back in the log.
+alias conda-create-env='conda env create --name'    # Creates a conda enviroment with the name as first argument($1)
+alias conda-remove-env='conda env remove --name'    # Removes a conda enviroment with the name as first argument($1)
+alias conda-eu="conda env update"                   # Updates the current conda environment
+alias conda-ls='conda env list'                     # List conda environments
+alias conda-lsp='conda list --name'                 # List conda packages                                          
+alias conda-i="conda install"                       # Installs a list of packages into a specified conda environment provided as arguments
+alias conda-r="conda remove"                        # Removes a list of packages from a specified conda environment provided as arguments
+
+#Git Aliases                                                
+
+alias git-graph="git log --graph --abbrev-commit --decorate --format=format:'%C(bold green)%h%C(reset) - %C(bold cyan)%aD%C(reset) %C(bold yellow)(%ar)%C(reset)%C(auto)%d%C(reset)%n''%C(white)%s%C(reset) %C(dim white)- %an%C(reset)' --all" # Simple and Beautiful way to display git log
```

Comment:
I quite like the plain git-graph.  Can we create a separate alias or add an color option to the alias?

Reply:
Alright will make it with a different name and restore the original alias. What should I name this one?

Reply:
Something along the lines of pretty? git-graph-pretty , pretty-git-graph...